adding_a_license_to_your_repository_v2.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: Adding-a-License-to-Your-Repository
+aspect: processes
+stages:
+  - “development”
+short_description: Every product should be covered by a license. It is your creative work, you own it and you should decide what other people can or cannot do with it.
+tags:
+  - license
+  - readme
+  - software documentation
+  - product
+  - documentation
+  - information
+  - description
+  - introduction
+  - installation
+  - writing
+keywords:
+  - license
+  - repository
+  - royalty terms
+  - copyright
+---
+
+# Adding a License to Your Repository
+
+## TL;DR

```

Comment:
```suggestion
**TL;DR**
```

use just bold for tl;dr pls

Reply:
@prokopsimek I can’t accept your suggestions because I don’t have permission to push to this repository